Classes/Property/TypeConverter/ExposedValueObjectConverter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Netlogix\JsonApiOrg\AnnotationGenerics\Property\TypeConverter;
+
+use Neos\Flow\Property\PropertyMappingConfigurationInterface;
+use Neos\Flow\Property\TypeConverter\AbstractTypeConverter;
+use Netlogix\JsonApiOrg\AnnotationGenerics\Cache\ExposedValueObjectCache;
+use Netlogix\JsonApiOrg\AnnotationGenerics\Domain\Model\ExposedValueObjectInterface;
+
+class ExposedValueObjectConverter extends AbstractTypeConverter
+{
+
+    protected $sourceTypes = ['array', 'string'];
+
+    /**
+     * The target type this converter can convert to.
+     */
+    protected $targetType = ExposedValueObjectInterface::class;
+
+    public function convertFrom(
+        $source,
+        $targetType,
+        array $convertedChildProperties = [],
+        PropertyMappingConfigurationInterface $configuration = null
+    ): ?ExposedValueObjectInterface {
+        return ExposedValueObjectCache::get($source, $targetType);
```

Comment:
I'd rather use a singleton instead of public static here. Currently cached data leaks at least from one test case to another.